modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java

```diff
@@ -52,6 +59,26 @@ public void testDeleteSaveWorkflowDefinition() throws Exception {
 			workflowDefinition.getVersion());
 	}
 
+	@Test
+	public void testDeployGroovyScriptBasedWorkflowDefinition()
```

Comment:
Do you think it's worth putting this method in some util class and reusing it here and in the **ObjectActionLocalServiceTest** and **ObjectValidationRuleLocalServiceTest** classes? We are replicating this method in the test classes.